metrics-delivery-endpoint-impl/src/main/java/eu/openaire/mas/delivery/mapping/MappingProvider.java

```diff
@@ -7,4 +7,6 @@
     PrometheusMetricMeta get(String groupId, String metricId) throws MappingNotFoundException;
     
     Set<String> listMetrics(String groupId) throws MappingNotFoundException;
+
+    Set<String> listGroups();
 }
```

Comment:
I guess we should align on the naming here as well:
* group -> resource
* metric -> kpi